From 0d136c7ab721be641cb97359c270ebe9e811007b Mon Sep 17 00:00:00 2001
From: Corentin LABBE <clabbe@baylibre.com>
Date: Fri, 28 Sep 2018 16:16:20 +0200
Subject: [PATCH 3/4] meson-gx-common: add booti support

This patch add booti support but with condition since some boards does
not support it.

Change-Id: I0603c8bd6e5b512f380860d5b6866d2e6729852d
---
 lava_scheduler_app/tests/device-types/meson-gx-common.jinja2 | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/lava_scheduler_app/tests/device-types/meson-gx-common.jinja2 b/lava_scheduler_app/tests/device-types/meson-gx-common.jinja2
index 46ce303a3..4d254a6fd 100644
--- a/lava_scheduler_app/tests/device-types/meson-gx-common.jinja2
+++ b/lava_scheduler_app/tests/device-types/meson-gx-common.jinja2
@@ -10,3 +10,10 @@
 {% set bootm_kernel_addr = bootm_kernel_addr|default('0x1080000') %}
 {% set bootm_dtb_addr = bootm_dtb_addr|default('0x1000000') %}
 {% set bootm_ramdisk_addr = bootm_ramdisk_addr|default('0x3000000') %}
+
+{% if use_booti %}
+{% set booti_kernel_addr = '0x1080000' %}
+{% set booti_dtb_addr = '0x1000000' %}
+{% set booti_ramdisk_addr = '0x3000000' %}
+{% endif %}
+
-- 
2.16.4

